kvm/src/lib.rs

```diff
@@ -219,11 +224,13 @@ impl Into<u64> for NoDatamatch {
 /// A wrapper around creating and using a VM.
 pub struct VmFd {
     vm: File,
+    #[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
```

Comment:
I would leave this target architecture configuration out for now because if we are adding it we should also propagate it to all places where the cpuid is used. For example we would need to also update the configure function of the Vcpu structure and the constructor (in vstate.rs). Same thing for the cpuid crate and probably others.

Reply:
Ok, I agree this changes are not related to this PR. I removed it. Take a look.